pyhf/commandline.py

```diff
@@ -5,12 +5,14 @@
 import click
 import json
 import os
+import jsonpatch
```

Comment:
We need the additional setup.py requirements for this dependency, no?

Reply:
added. schould be fine. only sub-dependency is jsonpinter which is in turn python-stdlib only